lib/kaguya/core.ex

```diff
@@ -128,6 +133,7 @@ defmodule Kaguya.Core do
     cancel_server_timer(state)
     socket = reconnect()
     send(self(), :init)
+    send(self(), :rejoin_chan)
```

Comment:
Yes, and besides I'm not sure there is a value in moving it into a separate task. i don't think there is anything else to do anything else besides sending out password and names.